huaweicloud/services/eip/resource_huaweicloud_eip_associate.go

```diff
@@ -37,18 +42,41 @@ func ResourceEIPAssociate() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-
-			"port_id": {
+			"public_ip": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-
-			"public_ip": {
+			"port_id": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
+				Computed: true,
 				ForceNew: true,
 			},
+			"fixed_ip": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Computed:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.IsIPAddress,
+				AtLeastOneOf: []string{"port_id"},
+				RequiredWith: []string{"network_id"},
+			},
+			"network_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
```

Comment:
suggest to add RequiredWith: []string{"fixed_ip"} beacuse network_id is useless when fixed_ip is not specified

Reply:
**ConflictsWith** has updated